flypg/launcher.go

```diff
@@ -79,6 +79,12 @@ func (l *Launcher) LaunchMachinesPostgres(ctx context.Context, config *CreateClu
 		client   = flyutil.ClientFromContext(ctx)
 	)
 
+	machineConf := l.getPostgresConfig(config)
+	// Can't use `hasRequiredMemoryForBackup` because of circular import.
+	if config.BackupsEnabled && machineConf.Guest.MemoryMB < 512 {
```

Comment:
Hrmm, so this is a little weird since the 1x shared VM setup can actually be scaled up to 2Gib.

I think if we are going to error, we may want to consider offering a pre-configured plan that comes with 512mib of memory.  Otherwise, we could just log a warning.

Reply:
> Hrmm, so this is a little weird since the 1x shared VM setup can actually be scaled up to 2Gib.

What's the relevance? Wouldn't the RAM check then detect the correct amount and pass, or is that not accurate here for some reason?